docs/HowTo/Deploy/Bootnodes.md

```diff
@@ -25,7 +25,7 @@ We recommend bootnode configuration is stored under source control.
 To allow for failure, specify all bootnodes on the command line (even to the bootnodes themselves). 
 
 !!! example 
-    If your network has 2 bootnodes, pass the following parameter to all nodes including the bootnodes. 
+    If your network has two bootnodes, pass the following parameter to all nodes, including the bootnodes. 
    
     `--bootnodes=enode://<publicKeyBootnode1>@10.0.0.100:30303, <publicKeyBootnode2>@10.0.1.101:30303`
```

Comment:
replace 10.0.0.100 with <ip_of_bootnode> & same for bootnode ip2

Reply:
the space after the comma should also be removed